keylime/cloud_verifier_tornado.py

```diff
@@ -399,6 +423,14 @@ def delete(self):
             logger.info("DELETE returning 404 response. agent id: %s not associated to this verifer.", agent_id)
             return
 
+        # Cleanup the cache when the agent is deleted. Do it early.
+        if agent_id in GLOBAL_POLICY_CACHE:
+            del GLOBAL_POLICY_CACHE[agent_id]
```

Comment:
Can you modify a map concurrently by multiple coroutines without having to lock?

Reply:
I was suspicious so I search a bit, but should not be strictly required for build-in data-types like set or dicts: https://docs.python.org/3/glossary.html#term-global-interpreter-lock

That is why I am a bit reluctant about the other PR. I counted 3 or 4 locks in one of the older reviews, that makes me clearly not smart enough to validate the code.

Reply:
That is what my own search on python documentation have shown (i.e., **not** strictly required for build-in data-types like set or dicts). Anyway, this was tested on a large(ish) deployment (low thousands of `agents`, tens of `verifiers`)